doc/rose-rug-advanced-tutorials-polling.html

```diff
@@ -142,15 +139,12 @@ <h3 class="alwayshidden">Runtime</h3>
       <pre class="prettyprint lang-cylc">
 [runtime]
     [[root]]
-        command scripting  = sleep 10
-        [[[event hooks]]]
-            succeeded handler = rose suite-hook
-            failed handler = rose suite-hook
+        script = sleep 10
     [[compose_letter]]
-        command scripting = sleep 5; echo 'writing a letter to Bob...'
+        script = sleep 5; echo 'writing a letter to Bob...'
     [[send_letter]]
         environment scripting = eval `rose task-env`
-        command scripting = """sleep 5;
+        script = """sleep 5;
                                echo 'Hello Bob' &gt; $ROSE_DATA/letter.txt;
```

Comment:
Can you adjust the whitespace on these following lines so they're inline with the above now shorter script = """ entry.